doc/devices/dts/device_tree.rst

```diff
@@ -80,6 +80,59 @@ This .fixup file by default resides in the board directory and is named
 dts.fixup.  This fixup file maps the generated include information to the
 current driver/source usage.
 
+.. _dt_vs_kconfig:
+
+Device Tree vs Kconfig
+**********************
+
+As mentioned above there are several tools used to configure the build with
+Zephyr.
+The two main ones, Device Tree and Kconfig, might seem to overlap in purpose,
+but in fact they do not. This section inteds to serve as a reference when
+deciding whether to place a certain configuration item in Device Tree or Kconfig.
+
+The scope of each configuration tool can be summarized as follows:
+
+* Device Tree is used to describe the **hardware** and its **boot-time
+  configuration**.
+* Kconfig is used to describe which **software features** will be built into
+  the final image, and their **configuration**.
+
+Hence Device Tree deals mostly with hardware and Kconfig with software. A
+noteworthy exception is Device Tree's `chosen` keyword, which allows the user
+to select a particular instance of a hardware device to be used for a concrete
+purpose by the software. An example of this is using this keyword to select
+a particular UART to be used as the system's console.
+
+In order to further clarify this separation, let's use a particular, well-known
```

Comment:
Delete the "In order":

    To further clarify this separation, ...

Reply:
Fixed